src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -116,42 +122,61 @@ public String getRepos() {
-        Token token = tokenDAO.findById(access_token);
-        if (token.getTokenSource().equals(TokenType.QUAY_IO.toString())) {
-            Optional<String> asString = ResourceUtilities.asString(TARGET_URL + "repository?public=false", token.getContent(), client);
-            
-            if (asString.isPresent()) {
-                RepoList repos = MAPPER.readValue(asString.get(), RepoList.class);
-                List<Container> containers = repos.getRepositories();
-                for (Container c : containers) {
-
-                    if (name == null ? (String) c.getName() == null : name.equals((String) c.getName())) {
-                        List<Container> list = containerDAO.findByNameAndNamespace(name, (String) c.getNamespace());
-
-                        if (list.isEmpty()) {
-                            long create = containerDAO.create(c);
-                            return containerDAO.findById(create);
-                        } else {
-                            System.out.println("Container already registered");
+            notes = "Register a container (public or private)",
+            response = Container.class)
+    public Container registerContainer(@QueryParam("container_name") String name, @QueryParam("enduser_id") Long enduserId) throws IOException {
+//        User user = userDAO.findById(userId);
+        List<Token> tokens = tokenDAO.findByEnduserId(enduserId);
+
+        for (Token token : tokens) {
+            if (token.getTokenSource().equals(TokenType.QUAY_IO.toString())) {
+                Optional<String> asString = ResourceUtilities.asString(TARGET_URL + "repository?public=false", token.getContent(), client);
+
+                if (asString.isPresent()) {
+                    RepoList repos = MAPPER.readValue(asString.get(), RepoList.class);
+                    List<Container> containers = repos.getRepositories();
+                    for (Container c : containers) {
+
+                        if (name == null ? (String) c.getName() == null : name.equals((String) c.getName())) {
+                            List<Container> list = containerDAO.findByNameAndNamespace(name, (String) c.getNamespace());
+
+                            if (list.isEmpty()) {
+                                c.setEnduserId(enduserId);
+                                long create = containerDAO.create(c);
+                                return containerDAO.findById(create);
+                            } else {
+                                System.out.println("Container already registered");
+                            }
                         }
-                    }
 
+                    }
+                } else {
+                    System.out.println("Received no repos from client");
                 }
-            }else {
-                System.out.println("Received no repos from client");
```

Comment:
There seem to be a lot of distracting whitespace changes. 
Please install the code formatting file https://seqware.github.io/docs/100-coding-standards/ it should make it easier to understand commits